main.tf

```
@@ -285,3 +285,9 @@ resource "aws_lambda_event_source_mapping" "this" {
     }
   }
 }
+
+resource "aws_lambda_function_url" "this" {
+  count              = local.create && var.create_lambda_function_url
```

```suggestion
  count = local.create && var.create_function && !var.create_layer && var.create_lambda_function_url

```

Also, added new line separator after the meta-argument (`count`).